assembl/static2/js/app/components/common/richTextEditor/index.jsx

```diff
@@ -83,6 +85,7 @@ export default class RichTextEditor extends React.Component<Object, RichTextEdit
       },
       () => {
         this.props.updateContentState(this.getCurrentRawContentState());
+        this.props.handleInputBlur();
```

Comment:
This will crash if handleInputBlur is null, see my comment above.

Reply:
Thanks for spotting this :) As of the solution I would prefer adding an if this function exists